pytorch_lightning/plugins/base_plugin.py

```diff
@@ -13,27 +13,26 @@
 # limitations under the License.
 import contextlib
 from abc import ABC, abstractmethod
-from typing import Any, Generator, Optional, overload, Sequence, Tuple
+from typing import Any, Callable, Generator, Optional, overload, Sequence, Tuple
 
 import torch
+from torch.nn import Module
 
 
 class Plugin(ABC):
     """Basic Plugin class to derive precision and training type plugins from."""
 
     @abstractmethod
-    def connect(self, model: torch.nn.Module, *args: Sequence,
-                **kwargs: Sequence) -> Optional[Tuple[torch.nn.Module, Sequence, Sequence]]:
+    def connect(
+        self,
+        model: Module,
+        *args: Sequence,
+        **kwargs: Sequence,
+    ) -> Optional[Tuple[Module, Sequence, Sequence]]:
```

Comment:
i know this is formatting so this is outside the scope of this PR, but is there a comment as to what this returns? Tuple[Module, Sequence, Sequence] isn't clear to me

Reply:
We will add docsttrkngs. Not sure, but here it's probably model, optimizers and schedulers